boards/fysetc/ucan/ucan.yaml

```diff
@@ -0,0 +1,16 @@
+identifier: ucan
+name: FYSETC UCAN
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
```

Comment:
```suggestion
```

https://github.com/zephyrproject-rtos/zephyr/pull/80466

Reply:
Thank you for reviewing! I've removed `xtools` from the list.